traced/ykrt/src/mt.rs

```diff
@@ -123,7 +129,7 @@ impl Drop for MT {
 /// The innards of a meta-tracer.
 struct MTInner {
     hot_threshold: AtomicHotThreshold,
-    active_threads: AtomicUsize,
+    active_user_threads: AtomicUsize,
```

Comment:
What's a user thread?

Broader question: should comment all of our struct and enum fields no matter how obvious it seems?

Reply:
Good point. Fixed in https://github.com/softdevteam/yk/pull/314/commits/a465e5ff4a3e3c48eed4934d64e72622fae5a01f.